h/static/styles/partials-v2/_search.scss

```diff
@@ -115,9 +115,10 @@
 // Card displaying stats about a group of annotations in search results
 .search-bucket-stats {
   @include font-normal;
-  width: 210px;
   margin-left: 30px;
   word-wrap: break-word;
+  min-width: 0;  // Make sure the .search-bucket-stats element's content wraps
```

Comment:
The comment explains what this line does but not the (IMO) much less obvious question of _how_. Given that we are pairing `word-wrap: break-word` and `min-width: N` declarations together in several elements here, it might be a good idea to extract the two into a mixin.

eg.

```scss
@mixin wrap-long-words(min-width: 0) {
  overflow-wrap: break-word;
  // Set an explicit min width, otherwise it will default to the
  // element's content size. See http://stackoverflow.com/a/36150764/434243
  min-width: $min-width;
}

.search-bucket-stats {
  @include wrap-long-words(0);
}
```